apps/heureka/src/components/services/ServicesListItem.jsx

```diff
@@ -9,11 +9,11 @@ import { listOfCommaSeparatedObjs } from "../shared/Helper"
 import constants from "../shared/constants"
 import { useGlobalsActions, useGlobalsShowPanel, useGlobalsShowServiceDetail } from "../StoreProvider"
 
-const countIssueMatches = (service) => {
-  return service?.componentInstances?.edges?.reduce((acc, edge) => {
-    return acc + (edge?.node?.issueMatches?.edges?.length || 0)
-  }, 0)
-}
+// const countIssueMatches = (service) => {
```

Comment:
I would propose to remove the commented functions since they can be found in the changes of this PR. Is this ok with you?